src/documents/ContextAwareness.md

```diff
@@ -0,0 +1,327 @@
+---
+title: ContextAwareness API
+layout: default
+category: API
```

Comment:
We need to make sure this page appears in the Table of Contents for this category.